WordPress/src/test/java/org/wordpress/android/ui/reader/viewmodels/ReaderPostDetailViewModelTest.kt

```diff
@@ -238,14 +238,14 @@ class ReaderPostDetailViewModelTest : BaseUnitTest() {
 
     /* SHOW POST - LOADING */
     @Test
-    fun `when show post is triggered, then loading state is shown`() = test {
-        val observers = init(showPost = false)
-        whenever(readerGetPostUseCase.get(anyLong(), anyLong(), anyBoolean())).thenReturn(Pair(readerPost, false))
+    fun `given local post not found, when show post is triggered, then loading state is shown`() =
```

Comment:
Suggestion (💡): How about including the `given local post found` as well to cover both scenarios? For example:

```
    @Test
    fun `given local post found, when show post is triggered, then loading state is not shown`() =
            test {
                val observers = init(showPost = false)

                viewModel.onShowPost(blogId = readerPost.blogId, postId = readerPost.postId)

                assertThat(observers.uiStates.first()).isNotInstanceOf(LoadingUiState::class.java)
            }
```

Reply:
Added in 9e56773

Reply:
Thank you @ashiagr , we are good to go! ✅